packit/command_runner.py

```diff
@@ -0,0 +1,99 @@
+import logging
+from typing import Callable
+
+from packit import utils
+from packit.actions import ActionName
+from packit.utils import run_command
+from packit.config import Config, PackageConfig
+from packit.local_project import LocalProject
+
+logger = logging.getLogger(__name__)
+
+
+class CommandRunner:
```

Comment:
I'm sorry but I don't follow: why would CommandRunner needed to know anything about actions, Config and PackageConfig?

In my head: command runner is a class which runs commands, and that's it, nothing else.

Reply:
My point here is that writing a CommandRunner with this design will result in a ton of boilerplate code and I'm not even sure how to do it properly.